aiorpc/exceptions.py

```diff
@@ -13,5 +13,13 @@ class RPCError(Exception):
     pass
 
 
+class EnhancedRPCError(Exception):
```

Comment:
Can we unify RPCError and EnhancedRPCError? It's a little bit confused.

Reply:
I left them separate because there are conditions where they can be thrown. Internal RPC Errors will throw an RPCError, while an exception thrown in the method will throw an EnhancedRPC Error. I don't have any issues making them the same, we'd just need to update the places where RPCError is thrown to reflect the new pattern